app/models/absrel.js

```diff
@@ -76,6 +76,7 @@ aBSREL.statics.spawn = function (fn, options, callback) {
 
   absrel.tagged_nwk_tree = options.nwk_tree;
   absrel.mail = options.mail;
+  absrel.source = "API";
```

Comment:
Is this right?

Reply:
Yes, ABSREL uses routes .invoke function to spawn jobs from the web. Like FEL.